data/text.js

```diff
@@ -732,7 +732,7 @@ exports.BattleText = {
 		removeItem: '#bugbite',
 	},
 	poltergeist: {
-		activate: "  [POKEMON] is about to be attacked by its [ITEM]!",
+		item: "  [POKEMON] is about to be attacked by its [ITEM]!",
```

Comment:
(Could use `addItem` perhaps, and avoid the special-case?)

Reply:
I think the fact that we're suggesting `addItem` clarifies that we're using the incorrect message type, and we should be using `activate` because no item is being added.